index.html

```diff
@@ -533,6 +584,63 @@
 
             </div>
         </div>
+
+        <div id="settings-modal" class="modal fade">
+            <div class="modal-dialog modal-lg">
+                <div class="modal-content">
+                    <div class="modal-header">
+                        <button type="button" class="close" data-dismiss="modal">&times;</button>
+                        <h4 class="modal-title">Settings</h4>
+                    </div>
+                    <div class="modal-body">
+                        <div class="split-container-left">
+                          <p>
+                            <b>
+                              Board and Pieces
+                            </b>
+                            <br/>
+                            <i>Changes to board and pieces will only take effect, once you watch or start a new game.</i><br/>
```

Comment:
I was worried here, because the piece_size also affects the position of the pieces. If the pieces are rescaled during a game, they are not going to be in the middle of the fields anymore. Are you sure you want them to be rescaled dynamically? It is definetly possible, but might go along with some issues.